### ISSUE TITLE
hoogle responds with 'Unknown flag: -e'

### ISSUE THREAD

**Issue by miguel-negrao** [atom-haskell/ide-haskell-hoogle#2]
hoogle version: hoogle-5.0.10

```
child_process.js:214 Uncaught (in promise) Error: Command failed: /home/miguel/.nix-profile/bin/hoogle search -vl -i -e --  (
Unknown flag: -e


    at ChildProcess.exithandler (child_process.js:214:12)
    at emitTwo (events.js:100:13)
    at ChildProcess.emit (events.js:185:7)
    at maybeClose (internal/child_process.js:850:16)
    at Socket.<anonymous> (internal/child_process.js:323:11)
    at emitOne (events.js:90:13)
    at Socket.emit (events.js:182:7)
    at Pipe._onclose (net.js:477:12)exithandler @ child_process.js:214emitTwo @ events.js:100emit @ events.js:185maybeClose @ internal/child_process.js:850(anonymous function) @ internal/child_process.js:323emitOne @ events.js:90emit @ events.js:182(anonymous function) @ net.js:477
```

```
hoogle --help
Hoogle 5.0.10, http://hoogle.haskell.org/

hoogle [COMMAND] ... [OPTIONS]

Commands:
  [search]  Perform a search
  generate  Generate Hoogle databases
  server    Start a Hoogle server
  replay    Replay a log file
  test      Run the test suite

Common flags:
  -? --help             Display help message
  -V --version          Print version information
     --numeric-version  Print just the version number
  -v --verbose          Loud verbosity
  -q --quiet            Quiet verbosity
```

**Comment by miguel-negrao** [comment]
```
hoogle [search] [OPTIONS] [QUERY]
  Perform a search

Flags:
     --colour --color   Use colored output (requires ANSI terminal)
  -l --link             Give URL's for each result
     --numbers          Give counter for each result
  -i --info             Give extended information about the first result
  -d --database=FILE    Name of database to use (use .hoo extension)
  -n --count=INT        Maximum number of results to return
  -r --repeat=INT       Number of times to repeat (for benchmarking)
Language:
     --haskell          Work with Haskell
     --frege            Work with Frege
     --compare=ITEM     Type signatures to compare against
```

**Comment by lierdakil** [comment]
Hi. Sorry about the belated reply. I just released v0.1.0 (which is basically a complete rewrite), which should support both hoogle-v4 and hoogle-v5, in addition to working with remote hoogle databases.

Please note that by default it's going to try to get to <http://hoogle.haskell.org>. If you want to use local database, change [**Hoogle Type** setting](https://atom-haskell.github.io/extra-packages/ide-haskell-hoogle/#choosing-remote-or-local-database)

**Comment by lierdakil** [comment]
P.S. Thanks for the report, btw!
